start_workers.py

```diff
@@ -18,7 +18,7 @@ def configure_logging():
 
 def start_worker(queues=None):
     if not queues:
-        queues = ['index', 're-index', 'index-zip']
+        queues = ['index', 'index-zip', 're-index']
```

Comment:
Should these use the settings instead of strings?

Reply:
done